frontend/src/metabase/public/components/EmbedFrame/LogoBadge.styled.tsx

```diff
@@ -1,26 +1,50 @@
 import styled from "@emotion/styled";
+import { css } from "@emotion/react";
 import ExternalLink from "metabase/core/components/ExternalLink";
 import { color } from "metabase/lib/colors";
 import { breakpointMinMedium } from "metabase/styled-components/theme";
 
-export const MetabaseLink = styled(ExternalLink)`
+export type Variant = "default" | "big";
```

Comment:
nit: `large`?

Reply:
What do you think about making this a bit more specific I'm not sure how much specific we want to make it either. e.g. `FooterVariant` or `EmbedFooterVariant`, but `Variant` sounds too generic, and even when importing it in the adjacent file, I doubt readers will have a clear picture of what this type is about unless looking its usage or declaration.